UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 12: ordinal not in range(128)

Hello.
I have been trying to run pronterface for a while now on Windows 10, I have the lastest binaries from the file Printrun-win-18Nov2017.zip but I had no luck. I get this message:

![image](https://user-images.githubusercontent.com/13765131/215303192-54d076c8-0bac-4fd5-b138-cc526116d3eb.png)

This is what the log prints each time:

```
Traceback (most recent call last):
  File "pronterface.py", line 66, in <module>
  File "printrun\pronterface.pyc", line 2294, in __init__
  File "wx\_core.pyc", line 7974, in __init__
  File "wx\_misc.pyc", line 6216, in SetInstallPrefix
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 12: ordinal not in range(128)
```

I tried deleting the ini config file from my AppData folder but that didn't change anything.

Thanks in advance!

Hi, thank you for replying so quickly!
I extracted and ran the exe inside the pronterface-windows-x64_3.8-2.0.0rc8 file and got this message:
![image](https://user-images.githubusercontent.com/13765131/215350308-9b88bb28-6b74-4d0c-ad32-42d0fcbf4ab7.png)
This time no log file was generated. There was a new ini config file on AppData but it was empty.

Running the nightly build yielded this message:
![image](https://user-images.githubusercontent.com/13765131/215350538-c640bda6-3392-4061-82e0-78e2f37dc08a.png)

```
Traceback (most recent call last):
  File "pronterface.py", line 62, in <module>
  File "printrun\pronterface.py", line 2582, in __init__
  File "printrun\pronterface.py", line 186, in __init__
wx._core.wxAssertionError: C++ assertion "strcmp(setlocale(0, 0), "C") == 0" failed at ..\..\src\common\intl.cpp(1699) in wxLocale::GetInfo(): You probably called setlocale() directly instead of using wxLocale and now there is a mismatch between C/C++ and Windows locale.
Things are going to break, please only change locale by creating wxLocale objects to avoid this!
```
My computer's region is Argentina in case it helps.

--- a/NEWS.md
+++ b/NEWS.md
@@ -1,3 +1,39 @@
+Printrun - 2.0.0
+====================
+
+Stable release supporting Python 3.10 and wxPython 4.2.
+
+See changes in previous pre-releases below for a full change log from previous
+stable release.
+
+### New Features
+
+ - Dropped support for Windows 32-bit executable (0c296ba)
+ - Added PrintrunGTK3 sub-module. Windows libraries (0c296ba)
+ - CairoSVG >= 2.6.0 not supported (0c296ba)
+ - Port to wxPython 4.2 (0c296ba)
+ - Pyglet >= 2.0 not supported (#1291)
+ - Added option for perspective transformation on gcode rendering (#1240)
+ - Port to Python 3.10 (#1224)
+ - Revised Pronterface graph color configuration (#1206)
+
+### Fixed Bugs
+
+ - Errors on Projector tool (0c296ba)
+ - Errors on Excluder tool (#1307)
+ - Unresponsive GUI (#1303)
+ - Icon/images not found. Wrong paths (#1285)
+ - Error on adjusting speed and flow rate (#1262)
+ - Fixed Armenian translation (#1244, #1245)
+ - Temperature graph empty or missing extruders (#1237)
+ - Princore runSmallScript incorrectly handled comments (#1163)
+
+### Administration
+
+ - Updated Windows and macOS CI/CD procedures
+ - Added CONTRIBUTORS.md to properly credit all the community (#1249)
+
+
 Printrun - 2.0.0rc8
 ====================
 

--- a/NEWS.md
+++ b/NEWS.md
@@ -8,6 +8,9 @@ stable release.
 
 ### New Features
 
+ - Windows binaries restricted to Python 3.10 only (#1311)
+ - Improved German translation (#1311)
+ - Updated translation templates (#1311)
  - Dropped support for Windows 32-bit executable (0c296ba)
  - Added PrintrunGTK3 sub-module (0c296ba)
    * Bundled Windows GTK3 libraries for the Projector tool
@@ -20,6 +23,7 @@ stable release.
 
 ### Fixed Bugs
 
+ - Invalid log file path crashed the Windows app (#1300)
  - Errors on Projector tool (0c296ba)
  - Errors on Excluder tool (#1307)
  - Unresponsive GUI (#1303)

--- a/NEWS.md
+++ b/NEWS.md
@@ -8,6 +8,7 @@ stable release.
 
 ### New Features
 
+ - Separate binaries provided for macOS versions 10, 11 and 12 (#1317)
  - Windows binaries restricted to Python 3.10 only (#1311)
  - Improved German translation (#1311)
  - Updated translation templates (#1311)
@@ -33,7 +34,7 @@ stable release.
  - Temperature graph empty or missing extruders (#1237)
  - Princore runSmallScript incorrectly handled comments (#1163)
 
-### Administration
+### Administrative
 
  - Updated Windows and macOS CI/CD procedures
  - Added CONTRIBUTORS.md to properly credit all the community (#1249)

--- a/NEWS.md
+++ b/NEWS.md
@@ -9,7 +9,8 @@ stable release.
 ### New Features
 
  - Dropped support for Windows 32-bit executable (0c296ba)
- - Added PrintrunGTK3 sub-module. Windows libraries (0c296ba)
+ - Added PrintrunGTK3 sub-module (0c296ba)
+   * Bundled Windows GTK3 libraries for the Projector tool
  - CairoSVG >= 2.6.0 not supported (0c296ba)
  - Port to wxPython 4.2 (0c296ba)
  - Pyglet >= 2.0 not supported (#1291)

--- a/printrun/printcore.py
+++ b/printrun/printcore.py
@@ -13,7 +13,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Printrun.  If not, see <http://www.gnu.org/licenses/>.
 
-__version__ = "2.0.0rc8"
+__version__ = "2.0.0"
 
 import sys
 if sys.version_info.major < 3:

